css-transforms-1/test/Overview.html

```diff
@@ -2576,9 +2576,9 @@ <h3 class="heading settled" data-level="7.3" id="svg-transform-functions"><span
-   <p>For the <code><a data-link-type="element" href="https://www.w3.org/TR/svg2/pservers.html#elementdef-pattern" id="ref-for-elementdef-pattern①">pattern</a></code> element, the <code><a data-link-type="element-sub" href="https://www.w3.org/TR/SVG11/pservers.html#PatternElementPatternTransformAttribute" id="ref-for-PatternElementPatternTransformAttribute⑤">patternTransform</a></code> attribtue and <a class="property" data-link-type="propdesc" href="#propdef-transform" id="ref-for-propdef-transform①⑨">transform</a> property define an additional transformation in the pattern coordinate system. See <code><a data-link-type="element-sub" href="https://www.w3.org/TR/SVG/pservers.html#PatternElementPatternUnitsAttribute" id="ref-for-PatternElementPatternUnitsAttribute①">patternUnits</a></code> attribute for details <a data-link-type="biblio" href="#biblio-svg2">[SVG2]</a>.</p>
-   <p>For the <code><a data-link-type="element" href="https://www.w3.org/TR/svg2/pservers.html#elementdef-linearGradient" id="ref-for-elementdef-linearGradient①">linearGradient</a></code> and <code><a data-link-type="element" href="https://www.w3.org/TR/svg2/pservers.html#elementdef-radialGradient" id="ref-for-elementdef-radialGradient①">radialGradient</a></code> elements, the <code><a data-link-type="element-sub" href="https://www.w3.org/TR/SVG11/pservers.html#LinearGradientElementGradientTransformAttribute" id="ref-for-LinearGradientElementGradientTransformAttribute⑤">gradientTransform</a></code> attribtue and <a class="property" data-link-type="propdesc" href="#propdef-transform" id="ref-for-propdef-transform②⓪">transform</a> property define an additional transformation in the gradient coordinate system. See <code><a data-link-type="element-sub" href="https://www.w3.org/TR/SVG/pservers.html#LinearGradientElementGradientUnitsAttribute" id="ref-for-LinearGradientElementGradientUnitsAttribute①">gradientUnits</a></code> attribute for details <a data-link-type="biblio" href="#biblio-svg2">[SVG2]</a>.</p>
-   <p>For the <code><a data-link-type="element" href="https://www.w3.org/TR/css-masking-1/#elementdef-clippath" id="ref-for-elementdef-clippath②">clipPath</a></code> element, the <a data-link-type="element-attr" href="https://www.w3.org/TR/SVG11/coords.html#TransformAttribute" id="ref-for-TransformAttribute①①">transform</a> attribtue and <a class="property" data-link-type="propdesc" href="#propdef-transform" id="ref-for-propdef-transform②①">transform</a> property define an additional transformation in the clipping path coordinate space. See <code><a data-link-type="element-sub" href="https://www.w3.org/TR/css-masking-1/#element-attrdef-clippathunits" id="ref-for-element-attrdef-clippathunits①">clipPathUnits</a></code> attribute for details <a data-link-type="biblio" href="#biblio-css-masking">[CSS-MASKING]</a>.</p>
```

Comment:
Please revert the changes here. They are not ending up into the final spec anyway.